src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Calculator from './Calculator';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return <Calculator />;
+  }
+}
```

Comment:
- [ ] [OPTIONAL] As this App is the root component, it will be much better if you put it directly in the src root

Reply:
@AnselemOdims thank you, I will do that.